ghost/core/core/server/services/email-service/EmailRenderer.js

```diff
@@ -247,11 +247,11 @@ class EmailRenderer {
         return locale;
     }
 
-    getFromAddress(post, newsletter) {
+    getFromAddress(post, newsletter, useFallback = false) {
         // Clean from address to ensure DMARC alignment
         const addresses = this.#emailAddressService.getAddress({
             from: this.#getRawFromAddress(post, newsletter)
-        });
+        }, {useFallbackAddress: useFallback});
```

Comment:
Why naming change from `useFallback` to `useFallbackAddress`? Pick one and stick with it

Reply:
Maybe this should be `useFallbackDomain`, since the implication is that the address should be generated to pass DMARC alignment when combined with _that domain_, instead of with the default bulk config domain